### PR TITLE
Warn when exploding portal projectiles would hit you

### PR DESCRIPTION
**Pull request by kitchen-ace**
853fe2e fixed this for harming allies, but not yourself.

### CODE REVIEW

**Comment by kitchen-ace** [comment]
I'm not sure about the confirmation string. It works fine for this (produces "Damnation Bolt") but maybe a better comparison and/or projectile name should be used.

File: crawl-ref/source/throw.cc [hunk 1]
```diff
@@ -854,6 +854,13 @@ bool throw_it(bolt &pbolt, int throw_2, dist *target)
                     monster *am = monster_at(*ai);
                     if (am)
                         cancelled = stop_attack_prompt(am, false, *ai);
+                    else if (*ai == you.pos())
+                    {
+                        string prompt = make_stringf("That %s is likely to hit you. Continue anyway?",
```
Comment (aidanholm) [inline crawl-ref/source/throw.cc]: Maybe just "That is likely to hit you. Continue anyway?" is sufficient?

Comment (kitchen-ace) [reply crawl-ref/source/throw.cc]: Sure, that works. Updated.

**Comment by aidanholm** [comment]
Thanks!

**Pull request merged**